web-client/extras/lib/Secret.ts

```diff
@@ -0,0 +1,154 @@
+import { CryptoUtils, Hash, PrivateKey } from '@nimiq/core';
+import { BufferUtils } from './BufferUtils';
+import { Entropy } from './Entropy';
+import { SerialBuffer } from './SerialBuffer';
+import { Serializable } from './Serializable';
+
+abstract class Secret extends Serializable {
```

Comment:
Would it be an idea to also port over the tests? For the `ExtendedPrivateKey`, `Secret`, etc.?

Reply:
I can try, but then we need a JS test-runner, too :/

Reply:
Added the relevant tests with `vitest` :muscle: